Source/CheatSheet/CQLBasics-cheatsheet.md

```diff
@@ -0,0 +1,121 @@
+# CQL Basics Cheat Sheet
+
+## Values
+
+|Value|Description|Example|
+|-----|-----------|-------|
+|Null|The null literal|<pre lang="cql">null</pre>|
+|Boolean|The boolean literals|<pre lang="cql">true, false</pre>|
+|Integer|Sequences of digits in the range 0..2<sup>31</sup>-1|<pre lang="cql">16, -28</pre>|
+|Long|Sequences of digits in the range 0..2<sup>63</sup>-1|<pre lang="cql">16000000000, -28000000000</pre>|
+|Decimal|Sequences of digits with a decimal point, in the range 0.0.. (10<sup>28</sup>-1)/10<sup>8</sup>|<pre lang="cql">100.015</pre>|
+|String|Strings of any character enclosed within single-ticks (')|<pre lang="cql">'pending', 'active', 'complete'</pre>|
+|Date|The at-symbol (@) followed by an ISO-8601 compliant representation of a date|<pre lang="cql">@2014-01-25</pre>|
+|DateTime|The at-symbol (@) followed by an ISO-8601 compliant representation of a datetime|<pre lang="cql">@2014-01-25T14:30:14.559&#13;@2014-01T</pre>|
+|Time|The at-symbol (@) followed by an ISO-8601 compliant representation of a time|<pre lang="cql">@T12:00&#13;@T14:30:14.559</pre>|
+|Quantity|An integer or decimal literal followed by a datetime precision specifier, or a UCUM unit specifier|<pre lang="cql">6 'gm/cm3'&#13;80 'mm[Hg]'&#13;3 months</pre>|
+|Ratio|A ratio of two quantities, separated by a colon (:)|<pre lang="cql">1:128&#13;5 'mg' : 10 'mL'</pre>|
+|Code|Construct consistent with the way terminologies are typically represented|<pre lang="cql">Code '66071002' from "SNOMED-CT" display 'Type B viral hepatitis'</pre>|
+|Concept|Construct to specify multiple terminologies used to code for the same concept|<pre lang="cql">Concept {&#13;  Code '66071002' from "SNOMED-CT",&#13;  Code 'B18.1' from "ICD-10-CM"&#13;} display 'Type B viral hepatitis'</pre>|
+|Tuple|Structured values that contain named elements, each having a value of some type|<pre lang="cql">define "Info":&#13;  Tuple {&#13;    Name: 'Patrick',&#13;    DOB: @2014-01-01,&#13;    Address: Tuple { Line1: '41 Spinning Ave', City: 'Dayton', State: 'OH' },&#13;    Phones: { Tuple { Number: '202-413-1234', Use: 'Home' } }&#13;  }</pre>|
```

Comment:
Might be helpful to point out that you can create Tuples that represent a FHIR resource:
```
FHIR.Extension {
    "url": FHIR.uri { "value": 'some-uri' },
    "value": FHIR.code {
      "value": 'some value'
    }
  }
```

Reply:
I figured that would be good to include in the advanced cheat sheet. Along with multi-source queries, let statements, etc...